ouroboros-network/src/Ouroboros/Network/Mux/Egress.hs

```diff
@@ -171,9 +171,6 @@ processSingleWanton pmss mpi md wanton cnt = do
       -- return data to send
       pure frag
     let sdu = MuxSDU (RemoteClockModel 0) mpi md (fromIntegral $ BL.length blob) blob
-    void $ write pmss sdu
+    void $ (write $ bearer pmss) sdu
```

Comment:
This is a hairy combo of `()`s and `$`. Simpler as
```
void $ write (bearer pmss) sdu
```

Reply:
Hmm, that's strange since I changed this; maybe something went untoicced while rebaseing.

Reply:
I remember this comment, and it was addressed somewhere in karkn/mux-ver :)

Reply:
I am afraid it's a :ghost: ;)